addons/base_import/tests/test_base_import.py

```diff
@@ -648,6 +648,69 @@ def test_newline_import(self):
 
         self.assertItemsEqual(data, [data_row])
 
+    def test_set_empty_value_import(self):
+        partners_before = self.env['res.partner'].search([])
+        import_wizard = self.env['base_import.import'].create({
+            'res_model': 'res.partner',
+            'file': """foo,US,person\n
+foo1,Invalid Country,person\n
+foo2,US,persons\n""",
+            'file_type': 'text/csv'
+        })
+
+        results = import_wizard.execute_import(
+            ['name', 'country_id', 'company_type'],
+            [],
+            {
+                'quoting': '"',
+                'separator': ',',
+                'import_set_empty_fields': ['country_id', 'company_type'],
+            }
+        )
+        partners_now = self.env['res.partner'].search([]) - partners_before
+        self.assertEqual(len(results['ids']), 3, "should have imported the first 3 records in full, got %s" % results['ids'])
+
+        self.assertEqual(partners_now[0].name, 'foo', "New partner's name should be foo")
```

Comment:
Actually those assert comments do not add much value. Either we have something to highlight to people, either we don't and in that case, I would just give no message.